oas_docs/output/kibana.yaml

```diff
@@ -19328,6 +19328,46 @@ paths:
 
         Refer to [Spaces](https://www.elastic.co/docs/deploy-manage/manage-spaces) for more information.
   /api/entity_store/engines:
+    delete:
+      operationId: DeleteEntityEngines
+      parameters:
+        - description: The entity type of the engine ('user', 'host', 'service', 'generic').
+          in: query
+          name: entityTypes
+          required: true
+          schema:
+            description: Array of engine types to delete. Defaults to an empty array.
```

Comment:
Might be useful to add that an empty array will result with all entity engines to be deleted.